### PR TITLE
Open port 85 in the qemu containers 

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: Title
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
@kacf if it please the lord

File: meta-mender-qemu/docker/qemux86-64/Dockerfile [hunk 1]
```diff
@@ -51,4 +51,5 @@ RUN if [ `stat -c %s ovmf.qcow2` -eq 0 ]; then rm -f ovmf.qcow2; fi
 
 RUN chmod +x entrypoint.sh mender-qemu
 EXPOSE 8822
+EXPOSE 85
```
Comment (kacf) [inline meta-mender-qemu/docker/qemux86-64/Dockerfile]: There is no port 80 here! [This](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-qemu/scripts/docker/entrypoint.sh#L41) is what needs to change.

**Comment by kacf** [comment]
No idea what's going on with the tests... Rerun?

**Comment by oleorhagen** [comment]
@kacf failed on different integration tests. For one test same error that showed up 12(!) times in the last run is present, in the other one, a leftover client from another test appears :/

**Pull request merged**